docs/standard/native-interop/interop-guidance.md

```diff
@@ -0,0 +1,247 @@
+---
```

Comment:
one new thing that I recently learned about SEO is that we try to remove dupe works from the path. So the file name here could be just guidance or best-practices.md.